src/einsteinpy/tests/test_symbolic/test_tensor.py

```diff
@@ -126,55 +136,63 @@ def test_Tensor_repr():
     machine_representation = repr(obj1)
     assert not "object at 0x" in machine_representation
 
-
+@xfail(raises=TypeError)
 def test_TypeError2():
     scht = schwarzschild_tensor().tensor()
+    
     # pass non str object
+    
     try:
         obj = Tensor(scht, config=0)
         assert False
     except TypeError:
         assert True
 
 
+@xfail(raises=TypeError)
 def test_TypeError3():
     scht = schwarzschild_tensor().tensor()
+    
     # pass string containing elements other than 'l' or 'u'
+    
     try:
         obj = Tensor(scht, config="al")
         assert False
     except TypeError:
         assert True
 
-
+@xfail
 def test_subs_single():
+    
     # replacing only schwarzschild radius(a) with 0
+    
     T = schwarzschild_tensor()
     a, c = symbols("a c")
     test_arr = T.subs(a, 0)
     assert simplify(test_arr.arr[0, 0] - 1) == 0
     assert simplify(test_arr.arr[1, 1] - (-1 / c ** 2)) == 0
 
-
+@xfail
```

Comment:
Again, unnecessary. Please remove the similar instances of `xfail`

We are trying to use `xfail` wherever we are expecting an `Exception` or `Error` and is being handled by try-except blocks to make them pass.

Reply:
done